### DISCUSSION TITLE
Decode UVJ file: the case when JSON settings/config has an empty `Layers` array

### DISCUSSION THREAD

**Discussion by Megidd** [sn4k3/UVtools#231] [Q&A] [answered]
Sometimes I'm observing this exception, could it be a bug? The exception is observed *occasionally* 😕 

The troublemaker file is attached too.

[128034414.zip](https://github.com/sn4k3/UVtools/files/6758101/128034414.zip)

![image](https://user-images.githubusercontent.com/17475482/124348431-20e5b300-dbff-11eb-892a-f75ad84cc434.png)

**Comment by sn4k3** [comment, marked as answer]
Ok that >= is wrong, should be >
But can layers array be empty?
Never saw a empty array before on uvj

**Reply by Megidd** [reply]
Thanks @sn4k3 ! 😄 

It would be great to fix it just to be sure 😃
